sdk/response.js

```diff
@@ -84,10 +84,7 @@ class Response extends Api {
    * @return {boolean}
    */
   hasReturn() {
-    return this
-      .getServiceSchema(this._name, this._version)
-      .getActionSchema(this._actionName)
-      .hasReturn();
+    return this.return !== undefined;
```

Comment:
The exception is cached already in userland. I'm talking about adding a try/catch on the SDK call to newResponse().